tests/test_convert_delta_to_json.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 MosaicML LLM Foundry authors
```

Comment:
move this file to tests/a_scripts (tests have been reorganized as of https://github.com/mosaicml/llm-foundry/pull/768)